src/assets/styles/styles.css

```diff
@@ -178,6 +190,18 @@ body {
 
     }
 
+    .main_intro_links_bar {
```

Comment:
personally I would suggest to move component related styles directly to the `.vue` file with this component. It is much easier to manage styles if all component related styles are inside this component file 😉

Reply:
ok thanks for the suggestions